src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -2127,10 +2107,10 @@ PeerImp::onMessage(std::shared_ptr<protocol::TMValidation> const& m)
             // peer receives within IDLED seconds since the message has been
             // relayed. Wait WAIT_ON_BOOTUP time to let the server establish
             // connections to peers.
-            if (app_.config().REDUCE_RELAY_ENABLE && (bool)relayed &&
-                (stopwatch().now() - *relayed) < squelch::IDLED &&
-                squelch::epoch<std::chrono::minutes>(UptimeClock::now()) >
-                    squelch::WAIT_ON_BOOTUP)
+            if (vpReduceRelayEnabled_ && (bool)relayed &&
+                (stopwatch().now() - *relayed) < reduce_relay::IDLED &&
```

Comment:
Random thought: these all-uppercase values are a little too much visual noise. Also, statements like `                reduce_relay::epoch<std::chrono::minutes>(UptimeClock::now()) > reduce_relay::WAIT_ON_BOOTUP` will be `false` and then permanently switch to `true`. It's almost better to have that as a separate gate.